overlord/servicestate/quota_control.go

```diff
@@ -99,13 +113,24 @@ func memoryCGroupEnabled() error {
 	}
 
 	// no errors so far but the only path here is the cgroups file without the memory line
-	return fmt.Errorf("cannot retrieve memory cgroup configuration, it is not available in the kernel config")
+	return fmt.Errorf("cannot retrieve %s cgroup configuration, it is not available in the kernel config", controllerName)
```

Comment:
```suggestion
	return fmt.Errorf("cannot retrieve %s controller configuration, it is not available in the kernel config", controllerName)
```

Reply:
The offending code is gone now after rebasing on latest master